PyInstaller/hooks/hook-PyQt5.QtQml.py

```diff
@@ -36,3 +38,481 @@
+        a_right = without_apos[1:-1]
+
+        # the main python file
+        self.analysis_file = a_right
+        if os.path.exists(self.analysis_file):
+            path_splits = os.path.split(self.analysis_file)
+            # the folder the main py file is in
+            self.analysis_folder = path_splits[0]
+            return True
+        else:
+            return False
+
+    def read_analys_file(self):
+
+        # This is where we read the main python file
+        # to find the main qml file used.
+
+        with open(self.analysis_file, mode='rb') as analys:
+            for line in analys:
+                if re.findall('[ ]+[#]', str(line)):
+                    # skip it is a comment in the code
+                    continue
+                else:
+                    # find line that contains a qml file
+                    for search_query in self.search_words:
+                        query = '.' + search_query + '[(].*?.*?.*?[)]'
+                        hyp_file = re.findall(query, str(line))
+
+                        if hyp_file:
+                            # we will use the first file
+                            first = hyp_file[0].replace(search_query, '')
+                            second = first[2:-1]
+                            if self.is_raw_string(second):
+                                return True
+                            else:
+                                return False
+                            break
+
+                        else:
+                            continue
+
+    def is_raw_string(self, raw):
+
+        # Here we read the line to see if there is
+        # a genuine qml filepath.
+        if re.findall('([+]|,|[)])', raw) or not re.findall('(\'|")', raw):
+            # It is a variable
+            return False
+
+        else:
+            if re.findall('qrc:', raw):  # it is a qrc
+                return False
+            else:
+                if raw[0] == '"':
+                    self.main_qml = raw.replace('"', '')
+                    return True
+
+                elif raw[0] == "'":
+                    self.main_qml = raw.replace("'", "")
+                    return True
+                else:
+                    return False
+
+    def reconstruct_qml_path(self):
+
+        # This is where we verify if the
+        # qml filepath is a relative path
+        # to the application's folder
+        # if it's not, we simply do nothing.
+        # it can't be a full path.
+        if os.path.exists(self.main_qml):
+            return False
+        else:
+            self.append_to_data(self.analysis_folder, self.main_qml)
+            self._first_handle(self.analysis_folder, self.main_qml)
+
+    def _crawl_qml_file(self, c_file):
+
+        # This is where we parse the qml file to look for
+        # import statements.
+
+        with open(c_file, mode='r', encoding='utf-8') as fh:
+            for line in fh:
+                if re.findall('[ ]+[//]', line):
```

Comment:
This is wrong in several ways:
- you are only interested if the line is a comment line. Thus no need to find all occurrences and thus no need for findall.
- you are not interested in what has been found, thus `re.match()` or `re.search()` would be more meaningful
- this will not match if the line starts with a tab character.
- the pattern *requires* a leading space
- this would also match ` / only one dash`

I suggest `if re.match(r'\s*//'):`